webpack.static-viz.config.js

```diff
@@ -15,7 +15,7 @@ module.exports = {
 
   entry: {
     "lib-static-viz": {
-      import: "./static-viz/index.js",
+      import: "./static-viz/index.ts",
```

Comment:
Is this change relevant to question caching?

Reply:
Nope, it's not intended, fixed